llvm/lib/Target/RISCV/RISCVInstrInfoZcmop.td

```diff
@@ -11,22 +11,60 @@
 //
 //===----------------------------------------------------------------------===//
 
+multiclass tmpEclass<bits<3> imm3> {
```

Comment:
Please do not add temporary code to a real target. You should write a unit test for your change. There many in llvm/test/TableGen/ already.

Reply:
Sure, in this PR, only the first commit is intended for merging into the main branch. The second commit serves merely as a helpful reference for the review process and is not meant for merging. I can promptly remove it if necessary. I provided a reference implementation on RISCV backend because I think that the testcases in the /TableGen directory may not adequately showcase the PR’s impact fully. Thank you.